modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -1,5 +1,5 @@
 [[tuning-changelogs]]
-= Tuning Changelogs
+= Tuning Changelogs in {productname}
```

Comment:
Please remove "in productname" from your titles. It is redundant information. A user should already know they are using the product.

Reply:
This has been added for the purpose of distinction of parent and child channels. We have discussed this separately in the chat today and will revisit at the squad meeting for final resolution.